src/gui/qgsmaptoolidentify.h

```diff
@@ -146,6 +147,13 @@ class GUI_EXPORT QgsMapToolIdentify : public QgsMapTool
      */
     QgsIdentifyMenu *identifyMenu() { return mIdentifyMenu; }
 
+    /**
+     * Converts point cloud identification results from variant maps to QgsMapToolIdentify::IdentifyResult and apply some formatting
+     * \note : the converted variant maps are pushed at the back of \a results without cleaning what's in it previously
+     * \since QGIS 3.18
+     */
+    static void fromPointCloudIdentificationToIdentifyResults( QgsPointCloudLayer *layer, const QVector<QVariantMap> &identified, QList<QgsMapToolIdentify::IdentifyResult> &results );
```

Comment:
I'd suggest marking this as SIP_SKIP -- it's not the kind of thing we want locked into the stable API pledge